<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_mdle</class>
 <widget class="QWidget" name="file_mdle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>配置文件名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_file_ini">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ui_file_ini_state">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>摄像头数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_dir_camera"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>日志文件名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_log_file"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>FPGA配置文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_fpga_file"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>指令运行路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_cmd_file"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>指令运行数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_cmd_data"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ui_read_set">
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_write_set">
         <property name="text">
          <string>写入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
